packages/@azure/servicebus/data-plane/README.md

```diff
@@ -1,22 +1,30 @@
-# @azure/service-bus
+# Azure Service Bus client library for Typescript and Javascript
 
-This library provides a convenient way to interact with [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/).
+Microsoft Azure Service Bus is a highly-reliable cloud messaging service between applications and services.
```

Comment:
"between applications and services" doesn't mean anything to me... is this copy from the service bus team?

Reply:
yes, its a copy paste :) :) I don't like it either. We can remove it. I am not attached to it

Reply:
Changed to

> Azure Service Bus is a highly-reliable cloud messaging service from Microsoft

We can update when we have brighter ideas